com.unity.render-pipelines.core/Editor/CoreEditorDrawers.cs

```diff
@@ -158,7 +158,13 @@ void IDrawer.Draw(TData data, Editor owner)
                     m_ActionDrawers[i](data, owner);
 
                 if (m_Anim != null)
+                {
                     CoreEditorUtils.EndAdditionalPropertiesHighlight();
+
+                    // While the highlight is being changed, force the Repaint of the editor
+                    if (m_Anim.value > -Mathf.Epsilon)
```

Comment:
we dont have something like "isZero-ish" in C#?

Reply:
I do not found any, in this case the editor was always being repainted, my mistake I hade the minus added.

Reply:
could we use https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html?

Reply:
ok, so that is checking if a value is between a tolerance of epsilon, might work. Thank you!!